src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -537,8 +538,10 @@ private bool SyncOptionalWorkloads()
                     Dictionary<string, string> owInstalledPkgs = new Dictionary<string, string>();  // packageId -> packageVersion
                     HashSet<string> owSyncRequestsPackageIds = new HashSet<string>();
                     TemplateLocator optionalWorkloadLocator = new TemplateLocator();
+                    Paths paths = new Paths(EnvironmentSettings);
+                    string dotnetPath = Path.GetDirectoryName(Path.GetDirectoryName(paths.Global.BaseDir));
 
-                    IReadOnlyCollection<IOptionalSdkTemplatePackageInfo> owPkgsToSync = optionalWorkloadLocator.GetDotnetSdkTemplatePackages(sdkVersion);
+                    IReadOnlyCollection<IOptionalSdkTemplatePackageInfo> owPkgsToSync = optionalWorkloadLocator.GetDotnetSdkTemplatePackages(sdkVersion, dotnetPath);
```

Comment:
I am doing end to end testing. This version has a "v" in front

Reply:
https://github.com/dotnet/sdk/blob/1e2d5ef1db6b7cfc6a33decc571c9f83ef09aecf/src/Cli/dotnet/commands/dotnet-new/NewCommandShim.cs#L84

Reply:
Need to change

`string dotnetPath = Path.GetDirectoryName(Path.GetDirectoryName(paths.Global.BaseDir));`
to
`string sdkVersion = EnvironmentSettings.Host.Version.Substring(1); // Host.Version (from SDK) has a leading "v" that need to remove.`

I don't have the permission to push to the branch. But after this change, it is good end to end. We need this change to be merge ASAP. So we can catch RC1 train